src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeModal.js

```diff
@@ -215,14 +215,16 @@ export const GcpComputeModalBase = ({
   onSuccess,
   currentRuntime,
   currentDisk,
+  isLoadingCloudEnvironments,
   tool,
   workspace,
   location,
   shouldHideCloseButton = true,
 }) => {
   // State -- begin
   const [showDebugger, setShowDebugger] = useState(false);
-  const [loading, setLoading] = useState(false);
+  const [_loading, setLoading] = useState(false);
+  const loading = _loading || isLoadingCloudEnvironments;
```

Comment:
Prefixing a variable name with an underscore is a convention for unused variables that are needed for destructuring, so this seems a bit misleading.